scripts/build/build-libkrunfw.sh

```
@@ -0,0 +1,119 @@
+echo "🔨 Building libkrunfw (this downloads kernel source on first run, ~10-20 min)..."
+cd "$LIBKRUNFW_SRC"
+make -j"$(nproc)" MAKEFLAGS=""
+
+# ── Stage the result ────────────────────────────────────────────────────────
+
+OUT_DIR="$(dirname "$OUT")"
+mkdir -p "$OUT_DIR"
+
+# libkrunfw's Makefile produces libkrunfw.so.5.<minor>.<patch> with a symlink
+# chain libkrunfw.so.5 → it. Copy the real file and stamp the requested SONAME.
+REAL_BLOB=$(ls "$LIBKRUNFW_SRC"/libkrunfw.so.5.* 2>/dev/null | head -1 || true)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)